execute/source_iterator.go

```diff
@@ -8,14 +8,14 @@ import (
 
 // SourceDecoder is an interface that generalizes the process of retrieving data from an unspecified data source.
 //
-// Connect implements the logic needed to connect directly to the data source.
+// connect implements the logic needed to connect directly to the data source.
 //
 // Fetch implements a single fetch of data from the source (may be called multiple times).  Should return false when
 // there is no more data to retrieve.
 //
 // Decode implements the process of marshaling the data returned by the source into a flux.Table type.
 //
-// In executing the retrieval process, Connect is called once at the onset, and subsequent calls of Fetch() and Decode()
+// In executing the retrieval process, connect is called once at the onset, and subsequent calls of Fetch() and Decode()
```

Comment:
Where are the calls of `Fetch()` and `Decode()` moved to? Or are they removed in this update?

Reply:
Oh, I see now. Sorry I was confused by `SourceDecoder` and `sourceDecoder` having such similar names and the renaming of those functions later in the file.